### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by tekktrik**
- Updates GitHub Actions step that attaches release assets to avoid deprecation
- Fixed botched metadata with stray link

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ inputs:
       The name for the CircuitPython repo.  The default is 'circuitpython-repo',
       and there typically isn't any reason to change this unless it conflicts
       with an existing folder name in your repository.
-    required: truehttps://github.com/adafruit/Adafruit_CircuitPython_turtle/pull/31
```
Comment (jepler) [inline action.yml:23]: that's .. interesting!

File: action.yml [hunk 2]
```diff
@@ -191,10 +191,11 @@ runs:
           rm $file
         done
     - name: Upload ZIP file to release
-      uses: csexton/release-asset-action@master
+      uses: shogo82148/actions-upload-release-asset@v1
```
Comment (jepler) [inline action.yml:194]: were you able to test this, or will it have to wait until a release is actually being made?

**Review by jepler — APPROVED** [review]
this all seems plausible but it might be best to actually merge this at a time that the relese action can be promptly tested and reverted if a problem pops up.

File: action.yml (hunk 2 above)
Comment (tekktrik) [reply action.yml:194]: Yes!  It's been tested on the libraries already.

File: action.yml (hunk 1 above)
Comment (tekktrik) [reply action.yml:23]: My touchpad seems to have a paste shortcut or something :sweat_smile:

**Pull request merged**